.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,15 +6,24 @@ labels: 'needs: discussion'
 assignees: ''
 
 ---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
 
-Those who maintain this project do so as volunteers and we have no control over their schedule or priorities.
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
 
-This means if the feature you are proposing is something you'd like to see added the best way to achieve that is for you to organise the effort required to build it. This could be evangelising for the feature, paying someone, doing it yourself, etc.
 
-Even so it might never get merged because the trade off between additional happy users and maintenance burden is not favourable.
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect who chooses to work on the feature with you. -->
+
+
+### How much effort will adding it take?
+<!-- Try to estimate how much work adding this feature will require. This information will affect who chooses to work on the feature with you. -->
```

Comment:
```suggestion
<!-- Try to estimate how much work adding this feature will require. Be as specific as you can (while recognising that its really hard to predict the future!). This information will affect who chooses to work on the feature with you. -->
```

Reply:
@betatim & @emdupre - Do you think this addition is useful? I like the point your making....but I wonder if the prompt is too long?

I'm very much on the fence with this, so just emoji vote this :+1: or :-1: (or suggest a different one!) and I'll go with what you think 👾

Reply:
"If in doubt shorter" wins for me on this one.

Reply:
Ok. Not going to merge this now. Let’s see how the prompt goes....